### ISSUE TITLE
dat link crashes

### ISSUE THREAD

**Issue by karissa** [maxogden/dat#435]
When there's a .git folder in the repo, and you run `dat link`
```
hyperdrive $ dat link
Creating share link for 15379 files, 3315 folders, 55.11 MB total
Fingerprinting file contents (22/15379)/Users/karissa/dev/node_modules/dat/node_modules/stream-each/index.js:28
      stream.destroy(err)
             ^

TypeError: stream.destroy is not a function
    at read (/Users/karissa/dev/node_modules/dat/node_modules/stream-each/index.js:28:14)
    at null.<anonymous> (/Users/karissa/dev/node_modules/dat/index.js:91:23)
    at emitOne (events.js:82:20)
    at emit (events.js:169:7)
    at Duplexify._destroy (/Users/karissa/dev/node_modules/hyperdrive/node_modules/pumpify/node_modules/duplexify/index.js:183:15)
    at /Users/karissa/dev/node_modules/hyperdrive/node_modules/pumpify/node_modules/duplexify/index.js:174:10
    at doNTCallback0 (node.js:419:9)
    at process._tickCallback (node.js:348:13)
```

**Comment by danfowler** [comment]
@karissa I ran into this when trying `dat` out on one of our datasets (which we tend to keep as their own repos)

**Comment by mafintosh** [comment]
fixed in 9.0
